config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -250,11 +250,6 @@
          files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]unnecessarysemicoloninenumeration.xml.template"/>
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]unusedlocalvariable.xml.template"/>
-  <!-- Properties macro suppressions until https://github.com/checkstyle/checkstyle/issues/13693 -->
-  <suppress id="propertiesMacroMustExist"
-         files="src[\\/]xdocs[\\/]checks[\\/]imports[\\/]customimportorder.xml.template"/>
-
-  <!-- no properties by design -->
```

Comment:
These 2 lines can't be removed unless you are saying these checks have properties now.

Reply:
done

Reply:
reverted, it is required to combine two parts of `no properties by design`, block above is same reason `no properties by design`